src/pages/AddPersonalBankAccountPage.tsx

```diff
@@ -103,6 +103,7 @@ function AddPersonalBankAccountPage({personalBankAccount, plaidData}: AddPersona
 AddPersonalBankAccountPage.displayName = 'AddPersonalBankAccountPage';
 
 export default withOnyx<AddPersonalBankAccountPageWithOnyxProps, AddPersonalBankAccountPageWithOnyxProps>({
+    // @ts-expect-error: ONYXKEYS.PERSONAL_BANK_ACCOUNT is conflicting with ONYXKEYS.FORMS.PERSONAL_BANK_ACCOUNT
```

Comment:
@blazejkustra Do you have any idea how we could solve this without expecting errors?

Reply:
For now it's the only way, keys are conflicting and the error is expected

Reply:
Yes, we should not conflicts onyx keys:
```ts
 /** Stores information about the active personal bank account being set up */
PERSONAL_BANK_ACCOUNT: 'personalBankAccount',
FORMS: {
        PERSONAL_BANK_ACCOUNT: 'personalBankAccountForm', // different than 'personalBankAccount'
        PERSONAL_BANK_ACCOUNT_DRAFT: 'personalBankAccountFormDraft',
}
```

PERSONAL_BANK_ACCOUNT and FORMS.PERSONAL_BANK_ACCOUNT should have different values. If not we would have to use ts-expect-error 😒

Reply:
> Do you think the first one is possible/reasonable?

It's possible, because we already have some keys that aren't unique. But this comes with a lot of ts-expect-errors in the code. We could overcome this by defining a common type for both personal bank account and personal bank account form, but this would require a big refactor across the codebase due to conflicts in these two types.

It's not reasonable, because with this we would compromise on types because a key could be either a `PersonalDetailsForm` type or `PersonalDetails` and this would create a big mess inside components (try it for yourself 😅)

I understand Expensify has other priorities at the moment, but the best approach would be to fix this on the backend side.